frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -2,7 +2,8 @@ import { Button, Container, Nav, Navbar, NavDropdown } from "react-bootstrap";
 import { Link } from "react-router-dom";
 import { hasRole} from "main/utils/currentUser";
 
-export default function AppNavbar({currentUser, doLogout}) { 
+export default function AppNavbar({currentUser, systemInfo, doLogout}) { 
+  console.log("systemInfo",systemInfo);
```

Comment:
Can we remove this log statement? Functional components should not have any side effects and this could clutter the log if React decides to rerender this component. If we need to observe the value of this prop in development we can always do that with React Dev Tools.